code/go/0chain.net/blobber/worker.go

```diff
@@ -102,42 +98,3 @@ func startRefreshSettings(ctx context.Context) {
 
 	}
 }
-
-func StartUpdateWorker(ctx context.Context, interval time.Duration) {
-	err := filestore.GetFileStore().CalculateCurrentDiskCapacity()
-	if err != nil {
-		panic(err)
-	}
-
-	for {
-		select {
-		case <-ctx.Done():
-			return
-		case <-time.After(config.Configuration.BlobberUpdateInterval):
-			err := filestore.GetFileStore().CalculateCurrentDiskCapacity()
-			if err != nil {
-				logging.Logger.Error("Error while getting capacity", zap.Error(err))
-				break
-			}
-			capacity := filestore.GetFileStore().GetCurrentDiskCapacity()
-
-			if uint64(config.Configuration.Capacity) != capacity {
-
-				_, err = config.ReloadFromChain(common.GetRootContext(), datastore.GetStore().GetDB())
```

Comment:
This will load the setting changes from network, if we remove this, the changes that done by blobber owner from zboxcli `bl-update` will never be synced. Btw, blobber could update the settings if blobber operation wallet is the delegate wallet.

Reply:
it is done on another startRefreshSettings worker.

Reply:
https://github.com/0chain/blobber/blob/3d22da9db16dce3bb4882063b189f8d970b75acd/code/go/0chain.net/blobber/worker.go#L75